### DISCUSSION TITLE
master code

### DISCUSSION THREAD

**Discussion by satstarter** [jellyfin/jellyfin-androidtv#2230] [General]
hi i have downloaded the master code and compiled in androidstudio
then installed on androidtv box the debug version so far so good
but when trying to play something i get error could not find item to play (itemId=null)
any sugestions? or is this normal?

thanx in advance

**Comment by nielsvanvelzen** [comment]
If you're experience a crash of different problem with the app please open an issue with reproducible steps/logs/etc.

**Reply by satstarter** [reply]
oke thanx for your reply i deleted everything and loaded the master a 30 min ago now the debug version
that i compiled is working but do you want that i report a bug thats in there because i tought because there's no release of it
its not necessary to open an issue yet
i compiled it to see whats new and not to report an issue yet
but if you want that i will do that i can test all day if necesarry ;)

forgive me for my english i'm dutch speaking

**Reply by nielsvanvelzen** [reply]
If it works now you don't need to create an issue.
